### DISCUSSION TITLE
Files on GitHub are limited to 1MB

### DISCUSSION THREAD

**Discussion by m3jobe** [jgraph/drawio#2578] [Q&A]
I am running into a size limit of 1MB when using github as storage

I see that this is mentioned here in the support page: https://drawio-app.com/github-support/

Is there any way to bypass this limit?

<img width="488" alt="Screen Shot 2022-01-25 at 1 07 11 PM" src="https://user-images.githubusercontent.com/61425024/151034034-23648229-153e-4a3c-b537-d028a24cdfc2.png">

**Comment by davidjgraph** [comment]
I think Github sets the limit. Are you sure this is github and not gitlab? If you have VScode you could use the extension there.

**Reply by m3jobe** [reply]
Yes Github Enterprise, what extension are you referring to for vscode?

**Reply by davidjgraph** [reply]
https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio
